src/test/fuzz/tx_pool.cpp

```diff
@@ -230,7 +239,9 @@ FUZZ_TARGET_INIT(tx_pool_standard, initialize_tx_pool)
         Assert(it->second.m_result_type == MempoolAcceptResult::ResultType::VALID ||
                it->second.m_result_type == MempoolAcceptResult::ResultType::INVALID);
 
-        const auto res = WITH_LOCK(::cs_main, return AcceptToMemoryPool(tx_pool, chainstate, tx, GetTime(), bypass_limits, /* test_accept= */ false));
+        chainstate.SetMempool(&tx_pool);
```

Comment:
Does this still work if this line is moved up to l145 (i.e. immediately after the mempool is constructed). I think the test is easier to understand if the mempool pointer is set at the beginning and not updated.